gax-java/gax/src/main/java/com/google/api/gax/rpc/ApiException.java

```diff
@@ -60,6 +60,18 @@ public ApiException(
     this.errorDetails = errorDetails;
   }
 
+  public ApiException(
```

Comment:
Is this used?

Reply:
Yeah, it'll be used in bigtable client. Do you want me to move it to a separate pr?

Reply:
Can you point me to how Bigtable would use it?

Reply:
Actually let me revert this and put in a separate PR.